css/properties/background-image.json

```diff
@@ -257,6 +257,40 @@
             }
           }
         },
+        "none": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "≤16"
```

Comment:
Seems safe to me to say that this was in Firefox 1. It is basic functionality of the property.